test/controllers/role_skill_controller_test.exs

```diff
@@ -1,83 +1,42 @@
 defmodule CodeCorps.RoleSkillControllerTest do
-  use CodeCorps.ApiCase
-
-  alias CodeCorps.RoleSkill
-  alias CodeCorps.Repo
-
-  @valid_attrs %{}
-  @invalid_attrs %{}
-
-  defp build_payload, do: %{ "data" => %{"type" => "role-skill", "attributes" => %{}}}
+  use CodeCorps.ApiCase, resource_name: :role_skill
 
   describe "index" do
     test "lists all entries on index", %{conn: conn} do
       path = conn |> role_skill_path(:index)
-      json = conn |> get(path) |> json_response(200)
-      assert json["data"] == []
+      conn = conn |> get(path)
+      assert json_response(conn, 200)["data"] == []
```

Comment:
does this work? i didn't realize there was a request_index function to help with that until just now.
```  
describe "index" do
    test "lists all entries on index", %{conn: conn} do
    [project_skill_1, project_skill_2] = insert_pair(:project_skill)

     conn
     |> request_index
     |> json_response(200)
     |> assert_ids_from_response([project_skill_1.id, project_skill_2.id])
    end
```